python_sdk/pyproject.toml

```diff
@@ -308,6 +308,13 @@ max-complexity = 17
 [tool.ruff.lint.per-file-ignores]
 
 
+"infrahub_sdk/client.py" = [
+    ##################################################################################################
```

Comment:
Can we get rid of the comment altogether here, it feels like these is nothing more to review :)

Reply:
It depends :)

I don't know if this is a bug in ruff, it counts the overload as an extra public method which might or might not be correct depending on your philosophy.

We can probably move away some of the public methods, for instance I don't know if .login really needs to be public and perhaps the ip stuff should live elsewhere. But we could also delete the comment for now, if ruff doesn't change this behaviour I don't think we'll get under 20 any time soon.